src/Integration.Vsix/SonarLintTagger/Scheduler.cs

```diff
@@ -0,0 +1,83 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading;
+
+namespace SonarLint.VisualStudio.Integration.Vsix
```

Comment:
@duncanp-sonar should I move this to another folder?
I wanted to move it to Core, but it needs the MEF decoration. I can move just the interface although that wouldn't do much.

Reply:
Leave it for now. We could make Core depend on MEF, but I've avoided doing that in the past because it didn't feel right.
One argument for moving the interface is that it makes it clear it's not dependent on VS (and also means we can't accidentally change the interface to depend on VS in a later refactoring). However, I don't think it matters too much in this case.